src/pages/orders/components/OrderDetailsCard/OrderDetailsCardInfo/OrderDetailsCardInfo.tsx

```diff
@@ -51,4 +53,11 @@ const OrderDetailsCardInfo = () => {
     );
 };
 
+const formatTestId = (text: string) => {
```

Comment:
Nice PR, just a small concern.

Let's remove the `formatTestId` method and simply use snake_case for all the `data-testid` attributes.

Reply:
Thank you for checking this. Actually I did this for order screen because there we have 6 labels that belong to a single div so giving a static data-testid would have applied same data-testid to all 6 labels. 

As data-testid needs to be unique, I am fetching the label headings and making is snake_case. (can be checked from screenshot)

One thing that can be done is move this formatTestId method in a common so that it can be called where it is required. 

![image](https://github.com/user-attachments/assets/7806bc9a-b093-44b0-95ce-c323bb6c9fe4)
![image](https://github.com/user-attachments/assets/254a0032-e901-42fa-9553-af3cbfa832af)